app/controllers/accounts/connected_accounts_controller.rb

```diff
@@ -16,7 +18,15 @@ def show
         sp_name: decorated_sp_session.sp_name,
         user: current_user,
         locked_for_session: pii_locked_for_session?(current_user),
+        all_emails_requested: all_emails_requested?,
       )
     end
+
+    def all_emails_requested?
```

Comment:
The method is named to suggest we're checking only `all_emails`, but we're also checking `email`.